BeVolt/BSP/Simulator/Src/BSP_Strobelight.c

```diff
@@ -0,0 +1,43 @@
+#include "../../Inc/BSP_Strobelight.h"
```

Comment:
You should just be able to put "BSP_Strobelight.h" in here. If you get an error from VSCode but not from the Makefile when you compile, it should be fine. The Makefile should have that directory on the include path.

Reply:
I would go ahead and change this to "BSP_Strobelight.h"